decentraland/helpers/storage.js

```diff
@@ -0,0 +1,59 @@
+import { SecureStore } from "expo";
+import { createFromPrivateKey } from "tasit-account/dist/testHelpers/helpers";
+
+// Storage keys
+const EPHEMERAL_ACCOUNT_PRIV_KEY = "EPHEMERAL_ACCOUNT_PRIV_KEY";
+const MY_ASSETS_LIST = "MY_ASSETS_LIST";
+
+export const storeEphemeralAccount = async account => {
+  const { privateKey } = account;
+  await _storeData(EPHEMERAL_ACCOUNT_PRIV_KEY, privateKey);
+};
+
+export const retrieveEphemeralAccount = async () => {
+  let account = null;
+
+  const privateKey = await _retrieveData(EPHEMERAL_ACCOUNT_PRIV_KEY);
+  if (privateKey != null) account = createFromPrivateKey(privateKey);
+
+  return account;
+};
+
+export const storeMyAssets = async myAssets => {
+  const strMyAssets = JSON.stringify(myAssets);
+  await _storeData(MY_ASSETS_LIST, strMyAssets);
+};
+
+export const retrieveMyAssets = async () => {
+  const strMyAssets = await _retrieveData(MY_ASSETS_LIST);
+  const myAssets = JSON.parse(strMyAssets);
```

Comment:
Always need a try/catch with things that throw like `JSON.parse`

Reply:
Done.

Reply:
Cool, thanks. Resolving